src/components/operations/operation-details/ko/runtime/graphql-console.html

```diff
@@ -27,87 +27,7 @@
-    <div class="row flex flex-row col-6">
-        <div class="col-4">
-            <label for="username" class="text-monospace form-label">Username</label>
-        </div>
-        <div class="col-6">
-            <div class="form-group">
-                <input type="text" id="username" class="form-control" data-bind="textInput: $component.username" />
-            </div>
-        </div>
-    </div>
-    <div class="row flex flex-row col-6">
-        <div class="col-4">
-            <label for="password" class="text-monospace form-label">Password</label>
-        </div>
-        <div class="col-6">
-            <div class="form-group">
-                <input type="password" id="password" class="form-control" data-bind="textInput: $component.password" />
-                <span class="invalid-feedback" data-bind="text: $component.authorizationError"></span>
-            </div>
-        </div>
-    </div>
-    <div class="row flex flex-row col-6">
-        <div class="col-4">
-        </div>
-        <div class="col-6">
-            <div class="form-group">
-                <button class="button button-primary"
-                    data-bind="click: $component.authenticateOAuthWithPassword">Authorize</button>
-            </div>
-        </div>
-    </div>
-    <!-- /ko -->
-    <!-- /ko -->
-
-    <!-- ko if: $component.subscriptionKeyRequired -->
-    <div class="row flex flex-row col-6 pad-left-0">
-        <div class="col-4 pad-left-0">
-            <label for="subscriptionKey" class="text-monospace form-label pad-top-12">
-                Subscription key
-            </label>
-        </div>
-        <div class="col-6">
-            <div class="form-group">
-                <!-- ko if: $component.products() && $component.products().length > 0 -->
-                <select id="subscriptionKey" class="form-control" data-bind="value: $component.selectedSubscriptionKey">
-                    <!-- ko foreach: { data: $component.products, as: 'product' } -->
-                    <optgroup data-bind="attr: { label: product.name }">
-                        <!-- ko foreach: { data: product.subscriptionKeys, as: 'subscriptionKey' } -->
-                        <option data-bind="value: subscriptionKey.value, text: subscriptionKey.name"></option>
-                        <!-- /ko -->
-                    </optgroup>
-                    <!-- /ko -->
-                </select>
-                <!-- /ko -->
-                <!-- ko if: !$component.products() || $component.products().length === 0 -->
-                <input id="subscriptionKey" type="text" class="form-control" placeholder="subscription key"
-                    data-bind="textInput: $component.selectedSubscriptionKey" />
-                <!-- /ko -->
-            </div>
-        </div>
-    </div>
-    <!-- /ko -->
+    <authorization params="{ api: $component.api, authorizationServer: $component.authorizationServer, gqlHeaders: $component.headers}"></authorization>
```

Comment:
`gqlHeaders` - looks like there is no (and should not be) such property in <authorization> component.